internal/config.go

```diff
@@ -143,3 +161,41 @@ func DefautConfigPath() string {
 	}
 	return filepath.Join(homeDirectoryPath, ".local/share/hz-cli", defaultConfigFilename)
 }
+
+func UpdateConfigWithSSL(config *hazelcast.Config, sslc *SSLConfig) error {
+	if sslc.ServerName != "" && sslc.InsecureSkipVerify {
+		return fmt.Errorf("SSL.ServerName and SSL.InsecureSkipVerify are mutually exclusive")
+	}
+	var tlsc *tls.Config
+	if sslc.ServerName != "" {
+		tlsc = &tls.Config{ServerName: sslc.ServerName}
+	} else if sslc.InsecureSkipVerify {
+		tlsc = &tls.Config{InsecureSkipVerify: true}
+	}
+	csslc := &config.Cluster.Network.SSL
+	if tlsc != nil {
```

Comment:
If a user sets `config.Cluster.Network.SSL` to false and `SSL.ServerName` to something so that `csslc.Enabled` is set to true? Is this documented? They may want to temporariliy disable SSL without deleting SSL part in yaml.

Reply:
The `SSL` configuration of CLC overrides Go client SSL config.

Reply:
It is ok to do that but we should improve documentation. Will you do later?